api/src/main/java/com/okta/sdk/resource/identity/provider/MicrosoftIdentityProviderBuilder.java

```diff
@@ -0,0 +1,14 @@
+package com.okta.sdk.resource.identity.provider;
+
+import com.okta.commons.lang.Classes;
+
+public interface MicrosoftIdentityProviderBuilder extends IdentityProviderBuilder<MicrosoftIdentityProviderBuilder> {
+
+    static MicrosoftIdentityProviderBuilder instance() {
+        return Classes.newInstance("com.okta.sdk.impl.resource.identity.provider.DefaultMicrosoftIdentityProviderBuilder");
+    }
+
+    MicrosoftIdentityProviderBuilder setIsProfileMaster(Boolean isProfileMaster);
```

Comment:
This field should be available for all types

Reply:
Noted.

Reply:
updated